website_blog_excerpt_img/README.rst

```diff
@@ -0,0 +1,68 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+   :alt: License: LGPL-3
+
+=======================
+Excerpt + Image in Blog
```

Comment:
Excerpt And Image in Blog

Reply:
@Nikul-Chaudhary Why? This file will be generated.